postgres/pgdump_test.go

```diff
@@ -783,7 +783,7 @@ func TestProcessPgDump_WithUnparsableContent(t *testing.T) {
 	if err == nil {
 		t.Fatalf("Expect an error, but got nil")
 	}
-	if !strings.Contains(err.Error(), "Error parsing") {
+	if !strings.Contains(err.Error(), "error parsing") {
```

Comment:
Is it more robust to do `!strings.Contains(strings.ToLower(err.Error()), "error parsing")`

Reply:
Good point. Done @ https://github.com/cloudspannerecosystem/harbourbridge/pull/172/commits/3701d42b42a0409b2981d46d8efa2c4f2ef60ba4.